webgl/lessons/ko/webgl-fundamentals.md

```diff
@@ -0,0 +1,610 @@
+제목: WebGL 기초
```

Comment:
The part before the colon's here are keywords. They need to be

```
Title: WebGL 기초
Description: 기초부터 시작하는 WebGL 첫 강의
```

Reply:
Oh I see, I will change it.